cpp/examples/sde_seir2v.cpp

```diff
@@ -0,0 +1,96 @@
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1)}]  = 100;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV2)}]  = 0.01;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV2)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1V2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::Susceptible)}] =
+        total_population -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1V2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}];
+
+    double beta = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(0.08), 0.5);
+    double kappa = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(8), 0.2);
+    double gamma = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(16), 0.2);
+
+    model.parameters.set<mio::sseir2v::TransmissionProbabilityOnContactV1>(beta);
+    model.parameters.set<mio::sseir2v::TransmissionProbabilityOnContactV2>(beta);
+    model.parameters.set<mio::sseir2v::TimeExposedV1>(kappa);
+    model.parameters.set<mio::sseir2v::TimeExposedV2>(kappa);      
+    model.parameters.set<mio::sseir2v::TimeInfectedV1>(gamma);
+    model.parameters.set<mio::sseir2v::TimeInfectedV2>(gamma);
```

Comment:
Is that right that both variants have the exact same parameter values?

Reply:
This is indeed not correct, "gamma" of the second Variant is assumed to be gamma / 1.35, the other parameters have the same values in both variants. I will check this and cite this from Simons paper